Octokit/GitHubClient.cs

```diff
@@ -100,6 +100,19 @@ public GitHubClient(IConnection connection)
         }
 
         /// <summary>
+        /// Set the timespan to wait before the requests to GitHub api times out.
+        /// Especially usefull to set a specific timeout that fit your needs to upload assets in a release
```

Comment:
Apart from the spelling mistake (useful has one L :)) I don't like the phrasing of the sentence sorry

How abou:

> /// Set the GitHub Api request timeout.
> /// Useful to set a specific timeout for lengthy operations, such as uploading release assets